Azure/azure-cli
# [Storage] BREAKING CHANGE: `az storage blob upload/upload-batch`: Fix `--overwrite`, no longer overwrite by default

**Description**

Used to overwrite everything by default. In storage-blob-preview extension, we introduced `--overwrite` to change this default behavior. 

**Testing Guide**

az storage blob upload-batch -d {containername} -s {filedirectory} 
#should show error
az storage blob upload-batch -d {containername} -s {filedirectory} 
#should succeed
az storage blob upload-batch -d {containername} -s {filedirectory} --overwrite

**History Notes**


[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change.
[Component Name 2] `az command b`: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
